components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -1519,12 +1517,11 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
             await this.teamSubscription2Service.cancelTeamMemberSubscription(ts2, userId, teamMembershipId, now);
         }
         const user = await this.userDB.findUserById(userId);
-        if (user && user.additionalData?.usageAttributionId === `team:${teamId}`) {
+        if (user && user.usageAttributionId === `team:${teamId}`) {
```

Comment:
There's helper logic to construct these IDs [here](https://github.com/gitpod-io/gitpod/blob/896f4b0f3e66aa053603c457bf82193cf833164e/components/server/src/user/user-service.ts#L247)